apps/dashboard/src/app/AppHeader.tsx

```diff
@@ -120,4 +154,43 @@ export class AppHeader extends React.Component<IAppHeaderProps> {
     }
     return true;
   }
+
+  private onFlightSelect(
+    field: keyof IAppSetting,
+    _ev?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>,
+    item?: IContextualMenuItem
+  ): boolean {
+    if (item?.data) {
+      // item.checked means it was checked before the click
+      if (item.checked) {
+        if (this.props.featureFlights.includes(item.data)) {
+          // need to remove flight
+          const previousFlights = parseFeatureFlights(
+            this.props.featureFlights
+          );
+          const newFlights = previousFlights.filter(
+            (flight) => flight !== item.data
```

Comment:
can you do _  != item.data here (?)

Reply:
oh I thought instead of (flight) => flight you can just use underscore.  Maybe I'm getting confused with scala here :sweat_smile:

Reply:
Underscore sort of indicates it's an arg that's not being used. Or lodash I guess?

I don't see an issue with this line of code as is.

Reply:
Resolving this for now, but feel free to ping me directly if you have further thoughts.